### DISCUSSION TITLE
React native list of variable types

### DISCUSSION THREAD

**Discussion by JulianHSC** [microsoft/react-native-windows#12950] [Q&A] [answered]
A google search doesn't seem to give any list. Can someone help? What types of variables are there and how do we create or declare them?

**Comment by chrisglein** [comment]
Please provide more detail to your question. Is this a question about JavaScript or JSX? Is there an aspect of your question that's about Windows and isn't covered by the documentation? As phrased this sounds like a question about what primitive types are possible in JavaScript.

**Comment by JulianHSC** [comment]
React native for mobile programming. I can't find it anywhere. I don't know what are primitive types. Can you help?

**Reply by chrisglein** [reply, marked as answer]
Try searching before asking? This was the first hit: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures
